packages/rules-unit-testing/package.json

```diff
@@ -20,10 +20,11 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "firebase": "8.6.5",
     "@firebase/component": "0.5.1",
     "@firebase/logger": "0.2.6",
     "@firebase/util": "1.1.0",
+    "firebase": "8.6.5",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
can you use a jose library instead? something like node-jose

Reply:
Any reason why? We use `jsonwebtoken` throughout the emulator suite and it's much more used than `node-jose` (200k download/mo vs 6m)

Reply:
I guess for emulator that's ok, the node-jose library also implements JWK which is helpful when you actually want to verify the signature, but you don't need that in this case